app/javascript/packages/document-capture/components/document-capture-troubleshooting-options.tsx

```diff
@@ -81,7 +83,13 @@ function DocumentCaptureTroubleshootingOptions({
         <TroubleshootingOptions
           isNewFeatures
           heading={t('idv.troubleshooting.headings.are_you_near')}
-          options={[{ text: t('idv.troubleshooting.options.verify_in_person') }]}
+          options={[
+            {
+              url: '#location',
```

Comment:
A bit of a cheat, but working as intended: Because FormSteps reflects the step as encoded in the URL hash fragment, we can link directly to the step, rather than using the submit button (`BlockSubmitButton`) to navigate as a submission of the step.

This was a workaround to an issue where `onClick` wasn't being called due to the fact that we were only spreading props on the link version of the option, not the button version:

https://github.com/18F/identity-idp/blob/93a5f0542d53656660132316b643191abf6e8ae6/app/javascript/packages/components/troubleshooting-options.tsx#L52-L56